### PR TITLE
Publish Machine API rate limits

### PR DESCRIPTION
**Pull request by kylemclaren**
Screenshot 2024-07-01 at 17 08 16

### Related Fly.io community and GitHub links

https://community.fly.io/t/understanding-fly-io-api-rate-hard-limits/20636

### CODE REVIEW

File: machines/api/working-with-machines-api.html.markerb [hunk 1]
```diff
@@ -73,4 +73,10 @@ Typically, 2xx HTTP status codes denote successful operations, 4xx codes imply f
 | 401 | The API key used was missing or invalid. |
 | 404 | The resource was not found. |
 | 408 | Request timed out. |
-| 5xx | Indicates an error with Fly.io API servers. |
\ No newline at end of file
+| 5xx | Indicates an error with Fly.io API servers. |
+
+## Rate Limits
+
+Machine API rate limits apply _per-action_, _per-machine_. Currently, the limit is 1 request, per second, per action (i.e. Create Machine, Start Machine etc.)—short-term burstable up to 3 req/s, per action.
```
Comment (andie787) [inline machines/api/working-with-machines-api.html.markerb:80]: Suggest removing "Currently" since it might imply an incoming change.

Comment (andie787) [inline machines/api/working-with-machines-api.html.markerb:80]: Suggest using "burst limit" in place of "burstable" here and below. I feel like burst limit pairs with rate limit and is a familiar term. What do you think?

**Review by andie787 — COMMENTED** [review]
Thank you for adding this missing info!!

Left a few suggestions. When you're happy with it, go ahead and merge.

**Review by catflydotio — COMMENTED** [review]
The content change looks great to me; I wouldn't make the change to the dashes earlier in the document, the argument being that that's effectively a style/formatting policy; i.e. a site-wide decision. We shouldn't change those in individual updates.

**Comment by kylemclaren** [comment]
@catflydotio Gotcha, how about removing the one em dash?

image

**Comment by andie787** [comment]
> The content change looks great to me; I wouldn't make the change to the dashes earlier in the document, the argument being that that's effectively a style/formatting policy; i.e. a site-wide decision. We shouldn't change those in individual updates.

No problem with the word. It's more about the context. Since we're sharing AWS links 😆 

https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-request-throttling.html

**Comment by kylemclaren** [comment]
Actually, I think you are right

**Pull request merged**